ReferenceQuality.pm

```diff
@@ -0,0 +1,194 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2018] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+ Ensembl <http://www.ensembl.org/info/about/contact/index.html>
+
+=cut
+
+=head1 NAME
+
+ ReferenceQuality
+
+=head1 SYNOPSIS
+
+ mv ReferenceQuality.pm ~/.vep/Plugins
+ ./vep -i variations.vcf --plugin ReferenceQuality,/path/to/data.gff3.gz
+
+=head1 DESCRIPTION
+
+ This is a plugin for the Ensembl Variant Effect Predictor (VEP) that
+ assesses the quality of the reference genome at the location of your variants.
+
+ The following steps are necessary before running this plugin:
+ 
+ GRCh38:
+ 
+ Download
+ ftp://ftp.ncbi.nlm.nih.gov/pub/grc/human/GRC/GRCh38/MISC/annotated_clone_assembly_problems_GCF_000001405.38.gff3
+ ftp://ftp.ncbi.nlm.nih.gov/pub/grc/human/GRC/Issue_Mapping/GRCh38.p9_issues.gff3
```

Comment:
It would be better to point to the latest version: GRCh38.p12_issues.gff3. This is the most up to date and the one we are using.